handlers/guestbook.go

```diff
@@ -405,7 +405,7 @@ func gbGeneralizer(data GbPreparedData) (map[string]interface{}, bool, string, e
 }
 
 func gbAddMessageSetCacheSettings(confirmedHtmxRequest bool) string {
-	cacheKey := "addGuestbookMessage"
+	cacheKey := "addMessage"
```

Comment:
The old cache key was preferable as the new one is too generic and may result in conflicts